### ISSUE TITLE
Notification text cannot be read when the window is resized down

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#44603]
VS Code version: Code - Insiders 1.21.0-insider (1a84a882737184488d965426747a05474f54a759, 2018-02-27T06:17:53.785Z)
OS version: Linux x64 4.13.0-36-generic

---

#44319

Repro:

1. Open a long notification
2. Resize the window to be very narrow, the text is cut off

![image](https://user-images.githubusercontent.com/2193314/36736903-940b0a5e-1b8e-11e8-9248-6b991b72bca8.png)

```ts
vscode.window.showInformationMessage('Veniam ullamco reprehenderit laboris anim labore cupidatat sunt. Aliqua aliquip laboris minim irure commodo ea nostrud amet. Eu cupidatat occaecat magna velit sit cupidatat ex eiusmod. Ex deserunt ad ut do elit nostrud pariatur est dolore anim non. Non dolore non consectetur esse.', 'e', 'f');
```

**Comment by bpasero** [comment]
This is very hard for me to solve because I cannot change the height of list items dynamically while the item is visible unless I would remove and add it again. 

One way out would be to always reserve a fixed height for the text and then let it have scrollbars...

/cc @joaomoreno

**Comment by joaomoreno** [comment]
@bpasero Yup there is no current way to do it unless you resplice the expanded elements in the list. Maybe it's not that expensive, can you try it?

**Comment by bpasero** [comment]
I can try, not for Feb though. The ugly part is that I basically have to do the entire layout myself.
